[bug-34] prefect ui 오류 수정 및 flow 2개(RandomForest, DecisionTree) 등록

lazychoi opened the pull request:
- prefect ui에 등록이 안 되는 문제 수정
- flow 2개 등록 : RandomForestClassifier, DecisionTreeClassifier
- agent 2개 등록

lazychoi committed:
[bug-34] prefect ui 오류 수정 및 2개 flow(RandomForest, DecisionTree) 등록

AstroNaut000 commented on routers.py:
```diff
@@ -27,18 +27,25 @@ def addData(request: Request, form_data: HotelForm = Depends(HotelForm.as_form))
+
+    model = mlflow.sklearn.load_model(f"models:/RandomForestClassifier/Production")
+    pred = model.predict(preprocessor)
+    ### 여기까지 테스트용
```
주석 부분은 삭제하는게 좋을 것 같습니다.

AstroNaut000 commented on train_flow.py:
```diff
@@ -11,7 +11,28 @@
+    df = get_data()
+    x, y, preprocessor, redis_vari_num = preprocessing(df)
+    log_preprocessor(preprocessor)
+    # model, params, model_name = set_model
```
삭제 하면 좋을 거 같습니다

AstroNaut000 reviewed (approved):
LGTM

pastnfast reviewed (approved):
LGTM

lazychoi committed:
[bug-34] redisai와 minio에서 불러온 artifact로 예측한 성능 비교

- routers.py: predict2 <- minio에서 불러온 artifact
- index.html: action="/predict" => redisai / action="/predict2" => minio

ISSUE #34

pastnfast reviewed (approved):
LGTM
